diff -Nur vert/value.go vert-new/value.go
--- vert/value.go	2022-08-07 11:59:18.144818494 -0700
+++ vert-new/value.go	2022-08-07 11:59:37.194524264 -0700
@@ -27,7 +27,7 @@
 // ValueOf returns the Go value as a new value.
 func ValueOf(i interface{}) Value {
 	switch i.(type) {
-	case nil, js.Value, js.Wrapper:
+	case nil, js.Value:
 		return Value{Value: js.ValueOf(i)}
 	default:
 		v := reflect.ValueOf(i)
